pootle/apps/pootle_store/migrations/0006_auto_20200610_2137.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.0.5 on 2020-06-10 21:37
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("pootle_store", "0005_auto_20200124_0617"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql=[
+                (
+                    """
+                CREATE PROCEDURE `insert_unique_submission` ( IN 
+                creation_time DATETIME, 
+                field_arg INT(11), 
+                submitter_id INT(11), 
+                type_arg INT(11), 
+                unit_id INT(11), 
+                store_id INT(11), 
+                translation_project_id INT(11), 
+                old_value longtext, 
+                new_value longtext, 
+                similarity double, 
+                mt_similarity double)
+
+                BEGIN 	
+                    DECLARE total_rows INT DEFAULT 0;      	
+
+                    SELECT COUNT(unit_id) INTO total_rows FROM `pootle_app_submission` WHERE `pootle_app_submission`.`creation_time` = creation_time AND `pootle_app_submission`.`field` = field_arg AND `pootle_app_submission`.`submitter_id` = submitter_id         AND `pootle_app_submission`.`type` = type_arg AND `pootle_app_submission`.`unit_id` = unit_id FOR UPDATE;
```

Comment:
Please reformat the query to avoid overly long lines. You can insert line breaks before `FROM`, `WHERE`, `AND`, etc.

Reply:
👌